version 0.17.1 having error when build

<!-- Please read the link below first. -->
<!-- https://github.com/ratson/cordova-plugin-admob-free/issues/161 -->
version 0.17.1 having error when build using adobe website

:compileReleaseJavaWithJavac
/Users/vincent/IonicCnt/KOMITID/platforms/android/src/name/ratson/cordova/admob/rewardvideo/RewardVideoListener.java:90: error: method does not override or implement a method from a supertype
    @Override
    ^

--- a/src/android/rewardvideo/RewardVideoListener.java
+++ b/src/android/rewardvideo/RewardVideoListener.java
@@ -86,8 +86,7 @@ class RewardVideoListener implements RewardedVideoAdListener {
         }
         executor.fireAdEvent("admob.rewardvideo.events.REWARD", data);
     }
-    
-    @Override
+
     public void onRewardedVideoCompleted() {
         executor.fireAdEvent("admob.rewardvideo.events.CLOSE");
     }

